ntbs-service/Pages/Notifications/Edit/_LabResults.cshtml

```diff
@@ -0,0 +1,62 @@
+@model ntbs_service.Models.Entities.CultureAndResistance
+
+@if (Model != null)
+{
+    <h2>Lab Results</h2>
+    <div class="notification-lab-results-summary" id="lab-results-summary">
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <dt class="notification-test-results-label"> Culture Positive </dt>
+                <dd class="cell-min-height"> @Model.CulturePositive </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <dt class="notification-test-results-label"> Species </dt>
+                <dd class="cell-min-height"> @Model.Species </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <dt class="notification-test-results-label"> Drug Resistance </dt>
+                <dd class="cell-min-height"> @Model.DrugResistanceProfile </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <dt class="notification-test-results-label"> Earliest Date </dt>
+                <dd class="cell-min-height"> @Model.FormattedEarliestSpecimenDate </dd>
+            </nhs-grid-column>            
+        </nhs-grid-row>
+        <nhs-grid-row>

```

Comment:
You  don't need these two lines - the columns will wrap around correctly based on their size (ditto the other pages)

Reply:
The problem with this was that it wasn't wrapping properly when one of the columns was bigger than others